lang/de.xml

```diff
@@ -1,20 +1,265 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <lang>
+	<main_menubar>
+		<file name="Datei">
+			<open_pref_path>Öffne Präferenzen-Pfad...</open_pref_path>
+			<install_firmware>Installiere Firmware</install_firmware>
+			<install_pkg>Installiere .pkg</install_pkg>
+			<install_zip>Installiere .zip, .vpk</install_zip>
+			<install_license>Installiere Lizenz</install_license>
+		</file>
+		<emulation name="Emulation">
+			<last_apps_used>Zuletzt benutzte Anwendungen</last_apps_used>
+		</emulation>
+		<configuration name="Konfiguration">
+			<settings>Einstellungen</settings>
+			<user_management>Benutzer</user_management>
+		</configuration>
+		<controls name="Steuerung">
+			<keyboard_controls>Tastatur</keyboard_controls>
+			<controllers>Controller</controllers>
+		</controls>
+		<help name="Hilfe">
+			<about>Über</about>
+			<welcome>Willkommen</welcome>
+		</help>
+	</main_menubar>
+
+	<app_context>
+		<update_history>Update-Verlauf</update_history>
+		<information>Information</information>
+		<app_delete>Diese Anwendung und alle damit verbundenen Daten, wie Nutzerdaten, werden gelöscht.</app_delete>
+		<save_delete>Möchtest du die diese Nutzerdaten löschen?</save_delete>
```

Comment:
> Either `Möchtest du die Nutzerdaten löschen?` or `Möchtest du diese Nutzerdaten löschen?`
Oopsie
> I'd also use `Spielstand` instead of `Nutzerdaten` `Möchtest` sounds a bit wrong as well in this case `Willst du den Spielstand löschen?` (Just a suggestion - your decision)
Even a stock PS Vita runs apps, not just games, hence "Nutzerdaten". If you say that "gespeicherte Daten" is better than "Nutzerdaten", we should change it to the adjective-noun wording